src/content/reference/react-dom/components/option.md

```diff
@@ -4,12 +4,16 @@ title: "<option>"
 
 <Intro>
 
-The [built-in browser `<option>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) lets you render an option inside a [`<select>`](/reference/react-dom/components/select) box.
+<div dir="rtl">
```

Comment:
Please don't add anything not existing in the English text like this div element